images/publisher/publish.sh

```diff
@@ -23,4 +27,7 @@ runtime_vars="{
   \"cluster_name\": \"$CLUSTER\"
 }"
 pubsub_message=`echo ${runtime_vars} | jq ". + ${METRIC_CONFIG}"`
-/root/google-cloud-sdk/bin/gcloud pubsub topics publish metrics-written --message "$pubsub_message"
+echo "Pubsub message: ${pubsub_message}"
+
+set -x
+gcloud pubsub topics publish metrics-written --message "$pubsub_message"
```

Comment:
doesn't need to be this PR but wanted to start you thinking about it: we might want a parallel pubsub for oneshot messages with a oneshot metrics handler listening on it. Then people could use oneshots ensure that their regression_test_config is WAI. So this topic name would need to be overrideable at the oneshot creation layer